modules/apps/foundation/configuration-admin/configuration-admin-api/src/main/java/com/liferay/configuration/admin/portlet/action/ConfigurationRenderCommand.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.portlet.action;
+
+import javax.portlet.RenderRequest;
+import javax.portlet.RenderResponse;
+
+/**
+ * Provides a common interface for all the render commands for the
+ * configurations in the System Settings portlet. A ConfigurationRenderCommand
+ * is responsible for rendering the configuration.
+ *
+ * Render commands should be defined as OSGi components and registered under the
+ * type <code>ConfigurationRenderCommand</code>:
+ * <pre>
+ * {@literal @}Component
+ * (
+ * 	  property={"configurationPid=com.foo.FooConfiguration"},
+ * 	  service = ConfigurationRenderCommand.class
+ * )
+ * public class MyCustomRenderCommand implements ConfigurationRenderCommand {
+ *
+ * }
+ * </pre>
+ *
+ * @author Pei-Jung Lan
+ */
+public interface ConfigurationRenderCommand {
```

Comment:
Why is this needed instead of just using MVCRenderCommand?

They seem almost the same (the only difference being "throws PortletException" in MVCRenderCommand, which makes me wonder why we don't have something similar here.

If we used the same name, we can still tell them apart by using different properties to register the component to associate it to a configuration instead of associating it to a portlet.

Reply:
@JorgeFerrer You're right. We can just use the MVCRenderCommand interface. I'll make the change.

Reply:
Cool, thanks!